docs/buildspecs/e4s_testsuite.rst

```diff
@@ -6,32 +6,26 @@ Buildtest E4S Testsuite Integration
 ===================================
 
 The `E4S Testsuite <https://github.com/E4S-Project/testsuite/>`_ is a collection of lightweight tests assembled for over 100 
-software products provided by `E4S <https://e4s.io>`_ deployments. 
+software products provided by Extreme Scale Scientific Software Stack `(E4S) <https://e4s.io>`_ deployments. 
 E4S is a spack based software stack but because not all spack packages provide internal testing functionality it 
 is useful to have an alternative framework for testing deployed software.
 
-The E4S Testsuite operates on a hierarchy of shell scripts with each test containing its own setup, clean, compile 
-and run scripts. The `top level driver script <https://github.com/E4S-Project/testsuite/blob/master/test-all.sh>`_ sets up the test environment based on a configuration file which sets 
+The E4S Testsuite operates on a hierarchy of shell scripts with each test containing its own ``setup.sh``, ``clean.sh``, ``compile.sh`` 
+and ``run.sh`` scripts. The `top level driver script <https://github.com/E4S-Project/testsuite/blob/master/test-all.sh>`_ sets up the test environment based on a configuration file which sets 
 compilers, mpi run commands and other environment specific parameters. It then iterates through the selected tests 
 and generates output consisting of log files for individual test runs and a summary listing of each test’s success or failure. The `README <https://github.com/E4S-Project/testsuite/blob/master/README.md>`_ provides more detail on its structure and usage.
 
-Tests in the testsuite typically include source code and build files needed to build a small test application against 
-libraries provided by the spack install along with any input files required for the subsequent run test. These are 
-often extracted from test functionality included in the product’s source 
-tree that is not included in the final install. An application may have multiple test definitions to support different 
-configurations such as the use of different hardware accelerators.  Products with spack packages that include spack test 
-functionality are trivial to add to the E4S Testsuite, as opposed to fully implementing a distinct test.
+The provided tests typically include source code, build, and input files for quick-running example applications. Some import test code from the spack install location or invoke spack internal tests directly.
```

Comment:
maybe format `spack install`. 

a reader may  not know what `spack internal tests` are so maybe do `spack internal tests (spack test)`